### ISSUE TITLE
[BITV] Missing label on `progress` element

### ISSUE THREAD

**Issue by susnux** [nextcloud-libraries/nextcloud-upload#1001]
All form elements need to have a label, including the `progress` element.

Moreover the progress element is not hidden from DOM when no upload is active, this causes issues with screen readers. Instead of using `max-width` something like `display: none` should be used.
